src/components/UiBreakpoints/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Theme, createTheme } from '@mui/material';
+
+const breakpointsTheme: Theme = createTheme({
+  breakpoints: {
+    values: {
+      xs: 374,
+      sm: 639,
+      md: 767,
+      lg: 1023,
```

Comment:
why not 1024? and other values looks strange

Reply:
For some reason, I can't make the other pixels look good, if you want, I'll change it to min-width and then the pixels will be normal